src/main/webroot/css/bucketeer-custom.css

```diff
@@ -0,0 +1,14 @@
+
+body {
+  font-family: Montserrat, sans-serif;
```

Comment:
I'd suggest using the font stack for Avante Garde here, after Montserrat: "Avant Garde", Avantgarde, "Century Gothic", CenturyGothic, AppleGothic, sans-serif; from this site: https://www.cssfontstack.com/Avant-Garde

Reply:
Montserrat was selected because that's what redoc uses. I don't think we can easily tweak Montserrat to have those fallbacks. Do you still think we should do this knowing that? If so, I can add.

Reply:
The documentation page now shares the look and feel of the upload form and the response pages (for more context).